### PR TITLE
Update dummy data to reflect individual projects and class projects

### LINKED ISSUE #165: [General] Add realistic dummy data for v0.1.0
AC: Revise dummy data for models and jobs to make it realistic. Use FlaskML projects from the collection of individual projects in 596E, as well as long term group projects. Use details from their projects such that they can relate to the interface.

### PR DESCRIPTION
**Pull request by ShriramG24**
Closes #165.

Models Page:
image

Small Block Forensics -- Model Details Page:
image

Image Object Detection -- Model Details Page:
image

Jobs Page:
image

Registration Page (includes models other teams are working on, but haven't added model details yet for those):
image

### CODE REVIEW

File: src/main/database/dummy_data/jobs.ts [hunk 1]
```diff
@@ -1,90 +1,115 @@
...
+    modelUid: 'sbf-model',
+    startTime: new Date('2023-10-02T14:19:08Z'),
+    endTime: new Date('2023-10-02T14:19:49Z'),
     status: 'Failed',
     inputs: [
       {
-        path: '/input/path2',
-        path_type: 'temp',
+        path: 'F:/USB/Forensic/Input',
+        path_type: 'USB Drive',
```
Comment (atharvakale343) [inline src/main/database/dummy_data/jobs.ts:41]: The `path_type` is actually a misnomer. It's a custom key that will be used by the model devs when running inference from the frontend. For example, for SBF, the inputs array could look like:

```
[
  {
    path: "/path/to/input/data",
    path_type: 'Known Dataset'
  },
  {
    path: "/path/to/new/input",
    path_type: 'Target Dataset'
  },
]
```

Comment (ShriramG24) [reply src/main/database/dummy_data/jobs.ts:41]: Ah, that makes more sense. Will fix.

Comment (ShriramG24) [reply src/main/database/dummy_data/jobs.ts]: I have changed it to `path_key` instead of `path_type`.

**Review by atharvakale343 — APPROVED** [review]

**Comment by atharvakale343** [comment]
Looks great!

**Pull request merged**